broker/setup_broker/lvk/templates/bq_lvk_alerts_schema.json

```diff
@@ -0,0 +1,216 @@
+      "type": "TIMESTAMP"
+    },
+    {
+      "description": "GraceDB ID: [{T,M}]SYYMMDDabc. Example: MS181101abc",
+      "mode": "REQUIRED",
+      "name": "superevent_id",
+      "type": "STRING"
+    },
+    {
+      "description": "url to GraceDB",
+      "fields": [
+        {
+          "description": "",
+          "mode": "REQUIRED",
+          "name": "gracedb",
+          "type": "STRING"
+        }
+      ],
+      "mode": "REQUIRED",
+      "name": "urls",
+      "type": "RECORD"
+    },
+    {
+      "description": "Event information",
+      "fields": [
+        {
+          "description": "Time of event (UTC, ISO-8601), e.g. 2018-11-01T22:22:46.654Z",
+          "mode": "REQUIRED",
+          "name": "time",
+          "type": "TIMESTAMP"
+        },
+        {
+          "description": "Estimated FAR in Hz",
+          "mode": "REQUIRED",
+          "name": "far",
+          "type": "FLOAT"
+        },
+        {
+          "description": "true if trials factor × FAR < 1/month for CBC events, otherwise false | true if trials factor × FAR < 1/year for burst events, otherwise false",
+          "mode": "REQUIRED",
+          "name": "significant",
+          "type": "BOOLEAN"
+        },
+        {
+          "description": "List of detectors, e.g. ['H1', 'L1','V1'] whose data have been used by the online pipeline that has produced the preferred event for that particular superevent",
+          "mode": "REPEATED",
+          "name": "instruments",
+          "type": "STRING"
+        },
+        {
+          "description": "The contents of a sky map in a multi-order FITS format as a Base64-encoded string.",
+          "mode": "REQUIRED",
+          "name": "skymap",
+          "type": "STRING"
+        },
+        {
+          "description": "{AllSky,BBH,EarlyWarning,MDC}",
+          "mode": "REQUIRED",
+          "name": "search",
+          "type": "STRING"
+        },
+        {
+          "description": "CBC",
+          "mode": "REQUIRED",
+          "name": "group",
+          "type": "STRING"
+        },
+        {
+          "description": "{gstlal,MBTA,pycbc,spiir,CWB}",
+          "mode": "REQUIRED",
+          "name": "pipeline",
+          "type": "STRING"
+        },
+        {
+          "description": "time interval for which the signal is detected (above noise)",
+          "mode": "NULLABLE",
+          "name": "duration",
+          "type": "TIMESTAMP"
+        },
+        {
+          "description": "estimation of the frequency of the signal's main component",
+          "mode": "NULLABLE",
+          "name": "central_frequency",
+          "type": "INTEGER"
+        },
+        {
+          "description": "Probability, under the assumption that the source is not noise, that at least one of the compact objects was a neutron star, that the system ejected a non-zero amount of neutron star matter, and that at least one of the compact objects has mass in the range 3-5 solar masses, respectively",
```

Comment:
The description seems to just be a reiteration of the sub-columns.

It should probably describe the aggregate.